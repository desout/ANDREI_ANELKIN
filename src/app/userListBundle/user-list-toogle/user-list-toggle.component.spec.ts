import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { UserListToggleComponent } from './user-list-toggle.component';
import {NO_ERRORS_SCHEMA} from '@angular/core';
import {TestModule} from '../../Test.module';

describe('UserListToggleComponent', () => {
  let component: UserListToggleComponent;
  let fixture: ComponentFixture<UserListToggleComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [ ],
      schemas: [NO_ERRORS_SCHEMA],
      imports: [TestModule],
      providers: []
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(UserListToggleComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
