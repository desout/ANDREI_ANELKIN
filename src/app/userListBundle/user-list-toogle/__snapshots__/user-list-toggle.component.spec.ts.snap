// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserListToggleComponent should create 1`] = `
<div
  id="root0"
  ng-version="7.0.3"
>
  <div
    class="userListToggleContainer"
  >
    <div
      class="card-header"
    >
      <span>
        <div>
          <app-user-item>
            <div
              class="item"
            >
              <div
                class="leftSideInfo"
              >
                <span>
                  
                </span>
                <br />
                <span>
                  
                </span>
                <br />
                <span>
                  
                </span>
              </div>
              <div
                class="rightSideInfo"
              >
                <span>
                  
                </span>
              </div>
            </div>
          </app-user-item>
        </div>
        <div>
          <app-user-list-toogle-button>
            <div
              class="btn btn-secondary dropdown-toggle"
            />
          </app-user-list-toogle-button>
        </div>
      </span>
    </div>
  </div>
</div>
`;
